data/sql/updates/pending_db_world/rev_1495464409315344160.sql

```diff
@@ -0,0 +1,147 @@
+(2032,5543,0.02,1,0,1,1),
+(2032,5774,0.02,1,0,1,1),
+(2032,6045,0.02,1,0,1,1),
+(2032,6454,0.02,1,0,1,1),
+(2032,7085,0.02,1,0,1,1),
+(2032,7090,0.02,1,0,1,1),
+(2032,7360,0.02,1,0,1,1),
+(2032,7363,0.02,1,0,1,1),
+(2032,7364,0.02,1,0,1,1),
+(2032,7449,0.02,1,0,1,1),
+(2032,7450,0.02,1,0,1,1),
+(2032,7975,0.1,1,0,1,1),
+(2032,7992,0.0508056,1,0,1,1),
+(2032,8029,0.0508056,1,0,1,1),
+(2032,8385,0.0508056,1,0,1,1),
+(2032,8386,0.0580636,1,0,1,1),
+(2032,8387,0.0725795,1,0,1,1),
+(2032,9293,0.0435477,1,0,1,1),
+(2032,10300,0.1,1,0,1,1),
+(2032,10301,0.1,1,0,1,1),
+(2032,10302,0.0653215,1,0,1,1),
+(2032,10312,0.0290318,1,0,1,1),
+(2032,10424,0.02,1,0,1,1),
+(2032,10601,0.02,1,0,1,1),
+(2032,10603,0.0362897,1,0,1,1),
+(2032,10604,0.0653215,1,0,1,1),
+(2032,10606,0.1,1,0,1,1),
+(2032,11098,0.02,1,0,1,1),
+(2032,11164,0.02,1,0,1,1),
+(2032,11165,0.02,1,0,1,1),
+(2032,11167,0.02,1,0,1,1),
+(2032,11202,0.0290318,1,0,1,1),
+(2032,11204,0.1,1,0,1,1);
+
+-- Atal'ai Artifacts (and 1 mithril deposit) are underground
+UPDATE `gameobject` SET `position_z`=-15.29  WHERE `guid`=30371;
+UPDATE `gameobject` SET `position_z`= 14.5   WHERE `guid`=30374;
+UPDATE `gameobject` SET `position_z`= 31.63  WHERE `guid`=30380;
+UPDATE `gameobject` SET `position_z`=-16.75  WHERE `guid`=30381;
+UPDATE `gameobject` SET `position_z`= 19.57  WHERE `guid`=30383;
+UPDATE `gameobject` SET `position_z`= 10.58  WHERE `guid`=30541;
+UPDATE `gameobject` SET `position_z`=-19     WHERE `guid`=30542;
+UPDATE `gameobject` SET `position_z`=-16.9   WHERE `guid`=30543;
+UPDATE `gameobject` SET `position_z`= 20.2   WHERE `guid`=30546;
+UPDATE `gameobject` SET `position_z`=-2.2    WHERE `guid`=30547;
+UPDATE `gameobject` SET `position_z`= 10.5   WHERE `guid`=30550;
+UPDATE `gameobject` SET `position_z`=-11.9   WHERE `guid`=30551;
+UPDATE `gameobject` SET `position_z`= 10.91  WHERE `guid`=30554;
+UPDATE `gameobject` SET `position_z`=-3.56   WHERE `guid`=30556;
+UPDATE `gameobject` SET `position_z`= 18.70  WHERE `guid`=30558;
+UPDATE `gameobject` SET `position_z`= 19.1   WHERE `guid`=30559;
+UPDATE `gameobject` SET `position_z`= 18.66  WHERE `guid`=30561;
+UPDATE `gameobject` SET `position_z`=-19     WHERE `guid`=30643;
+UPDATE `gameobject` SET `position_z`=-6      WHERE `guid`=30646;
+UPDATE `gameobject` SET `position_z`=-10.4   WHERE `guid`=30375;
+UPDATE `gameobject` SET `position_z`=-8.5    WHERE `guid`=30378;
+UPDATE `gameobject` SET `position_z`= 19     WHERE `guid`=31029;
+
+--Atal'ai Artifact twin spawn
+DELETE FROM `gameobject` WHERE `guid`=30593;
+DELETE FROM `gameobject` WHERE `guid`=30594;
+DELETE FROM `gameobject` WHERE `guid`=30587;
+
+-- Blue Pearls do not drop from giant clams outside of the Vile Reef in STV.
+DELETE FROM `gameobject_loot_template` WHERE `entry` IN ( 2954, 2959) and `item` = 4611;
+-- Blue Pearls do not drop from Small Barnacle Clam or Thick Shell Clam items.
+DELETE FROM      `spell_loot_template` WHERE `entry` IN (58168,58172) and `item` = 4611;
```

Comment:
DELETE FROM `spell_loot_template` WHERE `entry` IN (58168,58172) and `item` = 4611;

Mega spaces